src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs

```diff
@@ -151,16 +151,68 @@ public static object ConvertFromJson(string input, bool returnHashtable, out Err
         /// if the <paramref name="returnHashtable"/> parameter is true.</returns>
         [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", Justification = "Preferring Json over JSON")]
         public static object ConvertFromJson(string input, bool returnHashtable, int? maxDepth, out ErrorRecord error)
+            => ConvertFromJson(input, returnHashtable, maxDepth, jsonDateKind: JsonDateKind.Local, out error);
```

Comment:
Shouldn't this be `JsonDateKind.Default`?

Reply:
Yes it should be, sorry was playing around with comparing the defaults and never fixed this up when finalizing on the values.